src/StreamedPart.php

```diff
@@ -180,11 +180,12 @@ public function __construct($stream)
      */
     public function isMultiPart()
     {
-        return ('multipart' === mb_strstr(
```

Comment:
Can you keep the inline structure and avoid using a variable ?

Reply:
I just did it :)

Reply:
I will mark the conversation as resolved, also thank you for responding so quickly.